airflow/www_rbac/templates/airflow/graph.html

```diff
@@ -22,9 +22,7 @@
 {% block head_css %}
 {{ super() }}
 <link rel="stylesheet" type="text/css"
-    href="{{ url_for('static', filename='dagre.css') }}">
```

Comment:
What's happened to this CSS file? Is it needed still?

Reply:
We had 2 .css files included in graph.html. dagre.css and graph.css, which makes no sense as both were serving the same purpose i.e., styling the DAG, so combined the css of both in graph.css now, and removed dagre.css (as graph.css makes more sense in graph.html, keeping the same name)